dockerfiles/cli/cli.sh

```diff
@@ -35,16 +35,16 @@ cli_init() {
   CODENVY_HOST=${CODENVY_HOST:-${DEFAULT_CODENVY_HOST}}
 
   if [[ "${CODENVY_HOST}" = "" ]]; then
-    info "Welcome to Codenvy!"
+    info "Welcome to $CHE_FORMAL_PRODUCT_NAME!"
     info ""
     info "We did not auto-detect a valid HOST or IP address."
-    info "Pass CODENVY_HOST with your hostname or IP address."
+    info "Pass ${CHE_PRODUCT}_HOST with your hostname or IP address."
```

Comment:
good catch.  Fixed.

On the other points:
1. Yes, I agree we should only define a single name "Codenvy" and then convert that to ALL CAPS, all lower.  I haven't done that in this update, but I agree this is probably a good improvement.

2. I can move the functions out of cmd_start().  I was thinking that if there were functions for reuse that were only used in a single command, that they would be better suited in the file that used them.  But I get your point and placed them back into the base location.